src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -931,6 +939,60 @@ private void createAndSetStatistics()
         findPaneOfTypeByID(COOKS_LABEL, Label.class).setLabelText(numberOfcooks);
     }
 
+    /**
+     * Creates several statistics and sets them in the townHall GUI.
+     */
+    private void createAndSetStatistics()
+    {
+        final int citizensSize = townHall.getColony().getCitizens().size();
+
+        Map<String, Integer> jobCountMap = new HashMap<String, Integer>();
+        for (@NotNull final CitizenDataView citizen : citizens)
+        {
+            int length = citizen.getJob().split("\\.").length;
+            String job = citizen.getJob().split("\\.")[length  - 1].toLowerCase();
+            System.out.println(job);
+            jobCountMap.put(job, jobCountMap.get(job) == null ? 1 : jobCountMap.get(job) + 1);
+        }
+        System.out.println(jobCountMap);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS106)